src/api.js

```diff
@@ -15,3 +15,13 @@ export const fetchSingleArticle = (article_id) => {
       return data.article;
     });
 };
+
+export const fetchCommentsByArticleID = (article_id) => {
+  return axios
+    .get(
+      `https://nc-news-portfolio.onrender.com/api/articles/${article_id}/comments`
```

Comment:
For a potential refactor here, I'd look at creating a custom instance of Axios as per the NC Notes.